### ISSUE TITLE
Rucio front/backends broken in outsource

### ISSUE THREAD

**Issue by ershockley** [XENONnT/straxen#975]
After #866 processing on OSG is failing. In the past we had a single rucio frontend that could then use either a local or remote backend, which was important for OSG processing because we sometimes use local (e.g. when processing on Expanse) and sometimes use remote backends (when processing on all other sites). Now that they are separate frontends, we get an error when trying to register both frontends because most of the time the local_rse is `None`: 

```
File "/opt/XENONnT/anaconda/envs/XENONnT_2022.03.5/lib/python3.8/site-packages/straxen/storage/rucio_local.py", line 38, in __init__
    self.path = self.local_prefixes[local_rse]
KeyError: None
```

**Comment by JoranAngevaare** [comment]
@ershockley , you could very easily fix this in outsource by checking that the `socket.getfqdn()` is in `local_rses` before setting the local front-end right?

If you want I could make a small patch but I think the simplest solution is on outsource

**Comment by ershockley** [comment]
Yeah that's what I'm going to do for now, I just thought it was a bit nicer before when we register a single frontend and that frontend figures out what backend(s) to use, rather than having to explicitly set 2 frontends. But the fix in #976 also works, it's effectively the same thing.

**Comment by JoranAngevaare** [comment]
Right, there were several reasons to split the two as I explained in detail in https://github.com/XENONnT/straxen/pull/866
